services/121-service/src/cronjob/cronjob.service.ts

```diff
@@ -189,8 +140,51 @@ export class CronjobService {
           name !== 'constructor'
         );
       })
-      // Filter out this method.
-      .filter((name) => name !== 'getAllMethodNames');
+      .filter((name) => name.startsWith('cron'));
     return methodNames;
   }
+
+  // If not initialized correctly no use running the cronjobs.
+  private async initService(cronjobName: string): Promise<void> {
```

Comment:
I would expect a function called 'initService' to run when the app starts up. Maybe it could be named differently or keep the headers and baseurl functionality in 2 functions?

Reply:
Renamed it to `prepareCronJobRun(cronjobName)`.